drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-businesscentral-client/src/main/java/org/drools/workbench/screens/scenariosimulation/businesscentral/client/editor/ScenarioSimulationEditorBusinessCentralWrapper.java

```diff
@@ -333,9 +388,9 @@ protected void setSaveEnabled(boolean toSet) {
     protected Promise<Void> makeMenuBar() {
         scenarioSimulationEditorPresenter.makeMenuBar(fileMenuBuilder);
         if (workbenchContext.getActiveWorkspaceProject().isPresent()) {
-            final WorkspaceProject activeProject = workbenchContext.getActiveWorkspaceProject().get();
+            final WorkspaceProject activeProject = workbenchContext.getActiveWorkspaceProject().orElseThrow(IllegalStateException::new);
             return projectController.canUpdateProject(activeProject).then(canUpdateProject -> {
-                if (canUpdateProject) {
+                if (Boolean.TRUE.equals(canUpdateProject)) {
```

Comment:
Is this really needed? Can `canUpdateProject` be `null`?

Reply:
@danielezonca In real cases I analyzed it's never null, I modifed it just because `canUpdateProject` is a `Boolean` and not a `boolean` and sonar suggested to change it.